### ISSUE TITLE
🐛 BUG: no source location of missing package

### ISSUE THREAD

**Issue by locriacyber** [snowpackjs/astro#1806]
### What package manager are you using?

yarn

### What operating system are you using?

Linux

### Describe the Bug

Astro should show which template file depend on which missing dependency.

This is helpful when removing a dependency, because astro will show all the files depend on it in the error.

### Steps to Reproduce

1. `npm init astro` using template Starter Kit
2. Choose no feature when creating project
3. Add `import 'nonexistentdependency'` to `src/component/Tour.astro` file
4. `yarn; yarn dev`
5. Error!

**Expected Behavior**
It should show the dependency chain starting from the missing dependency up to the top level file using that dependency.

**Actual Behavior**
```
> yarn dev
yarn run v1.22.15
$ astro dev
[17:44:33] [snowpack] Hint: run "snowpack init" to create a project config file. Using defaults...
/tmp/test/node_modules/esinstall/lib/entrypoints.js:175
        throw new Error(`Package "${dep}" not found. Have you installed it? ${depManifestLoc ? depManifestLoc : ''}`);
              ^

Error: Package "nonexistentdependency" not found. Have you installed it? 
    at Object.resolveEntrypoint (/tmp/test/node_modules/esinstall/lib/entrypoints.js:175:15)
    at PackageSourceLocal.buildPackageImport (/tmp/test/node_modules/snowpack/lib/cjs/sources/local.js:424:40)
    at async PackageSourceLocal.prepare (/tmp/test/node_modules/snowpack/lib/cjs/sources/local.js:227:13)
    at async startServer (/tmp/test/node_modules/snowpack/lib/cjs/commands/dev.js:240:9)
    at async createSnowpack (file:///tmp/test/node_modules/astro/dist/runtime.js:283:14)
    at async createRuntime (file:///tmp/test/node_modules/astro/dist/runtime.js:303:7)
    at async dev (file:///tmp/test/node_modules/astro/dist/dev.js:17:19)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by jonathantneal** [comment]
This works much better in v0.21.

```
Error when evaluating SSR module /@fs/home/projects/nbwigaybq.github/src/pages/index.astro:
Error: Failed to resolve import "nonexistentdependency" from "src/components/Tour.astro". Does the file exist?
```

Reproduction: https://stackblitz.com/edit/github-equwn6?file=src/components/Tour.astro

**Comment by FredKSchott** [comment]
Thanks for the report! We believe that this has been fixed in astro v0.21, going out today. Let us know if you can still reproduce the issue after updating to v0.21 and we'll be happy to take a look.
